scripts/test.canisters.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+POCKET_IC_SERVER_VERSION=6.0.0
+POCKET_IC_SERVER_PATH="pocket-ic"
+PREVIOUS_II_WASM_PATH="internet_identity_previous.wasm.gz"
+PREVIOUS_ARCHIVE_WASM_PATH="archive_previous.wasm.gz"
+
+if [[ $OSTYPE == "linux-gnu"* ]] || [[ $RUNNER_OS == "Linux" ]]; then
```

Comment:
Did you intend this to be used by CI too?
I like that idea, but in that case it would be great if you actually used it in CI too. 😉

Reply:
Yes, that's the idea but I'll do it in another PR.